src/DataGrid.tsx

```diff
@@ -448,14 +448,15 @@ function DataGrid<R, SR, K extends Key>(
 
   useImperativeHandle(ref, () => ({
     element: gridRef.current,
-    scrollToColumn,
-    scrollToRow(rowIdx: number) {
-      const { current } = gridRef;
-      if (!current) return;
-      current.scrollTo({
-        top: getRowTop(rowIdx),
-        behavior: 'smooth'
-      });
+    scrollToCell({ idx, rowIdx }) {
+      const scrollToIdx =
+        idx !== undefined && idx > lastFrozenColumnIndex && idx < columns.length ? idx : undefined;
+      const scrollToRowIdx =
+        rowIdx !== undefined && isRowIdxWithinViewportBounds(rowIdx) ? rowIdx : undefined;
+
+      if (scrollToIdx !== undefined || scrollToRowIdx !== undefined) {
+        setScrollToPosition({ idx: scrollToIdx, rowIdx: scrollToRowIdx });
```

Comment:
If you want, an alternative could be:
- create a div with the same styles
- append it to the grid
- scroll to it
- immediately remove it

Reply:
I was thinking the same but I need to set [styles ](https://github.com/adazzle/react-data-grid/pull/2851#discussion_r1204541214)on the grid also. May be more work to set the previous styles after removing div. Wdyt?

Reply:
Makes sense